src/stylesheets/containers/config.scss

```diff
@@ -0,0 +1,74 @@
+@import '../variables.scss';
+
+.config-container {
+  position: fixed;
+  top: 20px;
+  left: 20px;
+  right: 20px;
+  width: auto;
+  height: 90vh;
+  background: #222231;
+  padding: 20px;
+  z-index: 1000;
+  .header {
+    display: flex;
```

Comment:
Probably we can come up with mixins
`@mixin center {
 justify-content:center;
align-items:center;
}`
`@mixin flex-row {
 display: flex;
flex-direction: row;
}`